anchor-contributor/Anchor.toml

```diff
@@ -37,3 +37,12 @@ program = "./tests/token_bridge.so"
 [[test.validator.account]]
 address = "3GwVs8GSLdo4RUsoXTkGQhojauQ1sXcDNjm7LSDicw19"
 filename = "./tests/token_config.json"
+
+[[test.validator.account]]
+address = "7UqWgfVW1TrjrqauMfDoNMcw8kEStSsQXWNoT2BbhDS5"
```

Comment:
Need the eth token bridge registered in local validator to create wrapped and complete transfer